pyproject.toml

```diff
@@ -49,33 +49,33 @@ djangorestframework = {git = "https://github.com/encode/django-rest-framework.gi
 djangorestframework-xml = "^2.0.0"
 feedparser = "^6.0.10"
 httplib2 = "^0.22.0"
-igraph = "^0.11.3"
+igraph = "^0.11.6"
 internetarchive = "^4.1.0"
 ipaddress = "^1.0.16"
 itypes = "^1.1.0"
-kombu = "^5.3.2"
-lxml = "^5.1.0"
+kombu = "^5.4.2"
+lxml = "^5.3.0"
 markdown2 = "==2.5.*"
 natsort = "^8.4.0"
 ndg-httpsclient = "^0.5.1"
-networkx = "^3.2.1"
+networkx = "^3.3"
 nose = "*"
-pandas = "^2.2.2"
+pandas = "^2.2.3"
 pillow = "*"
 pycparser = "^2.22"
 pyopenssl = "*"
-pyparsing = "^3.1.1"
+pyparsing = "^3.1.4"
 python = ">=3.12, <3.13"
-python-dateutil = "^2.8.1"
+python-dateutil = "^2.9.0"
```

Comment:
https://dateutil.readthedocs.io/en/stable/changelog.html

No changes, only bug fixes and features